### PR TITLE
UD-879 add body injector service

### PR DESCRIPTION
**Pull request by olexii4**
@benoitf, @ashumilova   WDYT?

### CODE REVIEW

File: src/components/body-injector/che-body-injector.service.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    // The parent of the new element
+    let parentElement = this.$document.find('body');
+    let itemElement = this.$compile(item)(angular.element(parentElement[0]).scope());
+    let itemId = itemElement.attr('id');
+    if(!itemId) {
+      return;
+    }
+
+    let oldItem = this.$document[0].getElementById(itemId);
+    if(oldItem) {
```
Comment (benoitf) [inline src/components/body-injector/che-body-injector.service.js]: miss space before (

File: src/components/body-injector/che-body-injector.service.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  /**
+   * Add item into the body
+   * @param item
+   */
+  addElement(item) {
+    // The parent of the new element
+    let parentElement = this.$document.find('body');
+    let itemElement = this.$compile(item)(angular.element(parentElement[0]).scope());
+    let itemId = itemElement.attr('id');
+    if(!itemId) {
```
Comment (benoitf) [inline src/components/body-injector/che-body-injector.service.js]: missing space before parenthesis

File: src/components/body-injector/che-body-injector.service.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  constructor($document, $compile) {
+    this.$document = $document;
+    this.$compile = $compile;
+  }
+
+  /**
+   * Add item into the body
+   * @param item
+   */
+  addElement(item) {
```
Comment (benoitf) [inline src/components/body-injector/che-body-injector.service.js]: maybe it should return true if successful add element, false if it was not able

File: src/components/body-injector/che-body-injector.service.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    parentElement.append(itemElement);
+  }
+
+  /**
+   * Remove item from the body
+   * @param item
+   */
+  removeElement(item) {
+    let itemId = item.attr('id');
+    if(!itemId) {
```
Comment (benoitf) [inline src/components/body-injector/che-body-injector.service.js]: space before (

File: src/components/body-injector/che-body-injector.service.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+      return;
+    }
+
+    let findElement = this.$document[0].getElementById(itemId);
+    if(findElement) {
```
Comment (benoitf) [inline src/components/body-injector/che-body-injector.service.js]: missing space before (

File: src/components/body-injector/che-body-injector.service.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+      oldItem.remove();
+    }
+    parentElement.append(itemElement);
+  }
+
+  /**
+   * Remove item from the body
+   * @param item
+   */
+  removeElement(item) {
```
Comment (benoitf) [inline src/components/body-injector/che-body-injector.service.js]: return false/true if successful/failure

**Comment by benoitf** [comment]
+1

File: src/components/body-injector/che-body-injector.service.js [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+/**
+ * This class is handling the service for the new element injection
```
Comment (ashumilova) [inline src/components/body-injector/che-body-injector.service.js]: can add here that element is injected to body tag

**Comment by ashumilova** [comment]
+1

**Pull request merged**